apps/public-docsite/src/components/Site/Site.tsx

```diff
@@ -57,13 +59,15 @@ export class Site<TPlatforms extends string = string> extends React.Component<
   };
 
   private _async: Async;
+  private _disposables: Function[];
```

Comment:
The disposables part is general cleanup (same functionality as before, better patterns)